### PR TITLE
Translated into fluent Japanese. 

### PR DESCRIPTION
**Pull request by haniokasai**
Translated into fluent Japanese.

### CODE REVIEW

**Comment by 64FF00** [comment]
@haniokasai Thanks a lot :D

File: resources/messages-jp.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
-
 # PurePerms by 64FF00 (xktiverz@gmail.com, @64ff00 for Twitter)
 
 # Japanese Translation by onebone (jyc00410@gmail.com)
-
+# 2015/08/03 translated into fluent Japanese by haniokasai (htek@haniokasai.com @haniokasai for Twitter)
```
Comment (64FF00) [inline resources/messages-jp.yml]: Thank you so much @haniokasai :D :3 :D :3

**Comment by haniokasai** [comment]
I'm very happy I could　translate pperms :)

**Comment by 64FF00** [comment]
@haniokasai <3 you

**Comment by 64FF00** [comment]
@haniokasai I still need to update config.yml to add your name though

**Comment by haniokasai** [comment]
oh,add my name to config.yml ? thanks you!

**Comment by 64FF00** [comment]
@haniokasai Yup :D

**Pull request merged**